### DISCUSSION TITLE
Restored the weather widget on Windows 10 19045.4096

### DISCUSSION THREAD

**Discussion by bbmaster123** [valinet/ExplorerPatcher#2865] [General]
![Untitled](https://github.com/valinet/ExplorerPatcher/assets/92948468/d9ea680b-8af8-4257-a2b0-2b3629526461)

okay this one took a few days to figure out
I had a feeling Microsoft changed something here, as the settings app now didn't show options for people on the taskbar, and also peopleband.dll in system32 was the last accessed dll as shown in process monitor. 

I tried lots of things but in the end, I ran vivetool's feature scanner on my windows 10 system, which took hours and downloaded around 38GB of symbols. Finally, I obtained a list of this build's hidden feature switches. I loaded the list into vivetool gui and searched for people. I was right. `DeprecatePeopleOnTaskbar: 47691186` set to enabled. turned that off and rebooted.

I had a people icon on my taskbar now, and the option appeared in settings, but no weather yet. I checked `Computer\HKEY_CURRENT_USER\SOFTWARE\ExplorerPatcher\twinui.pcshell` in regedit, and a lot of the values were still just 0 like before, so I deleted the folder and restarted explorer with the gui to let it redownload symobls. The weather widget finally showed up. mission accomplished.

Tl;dr
to fix on windows 10:
1. disable `47691186` with vivetool
2. Export explorerpatcher settings in about
3. Uninstall explorerpatcher from the gui, reboot
4. delete the contents of `Computer\HKEY_CURRENT_USER\SOFTWARE\ExplorerPatcher\` in regedit
5. Install explorerpatcher, import settings

In addition, I also found a code to reenable the timeline view everyone seems so desperate to have back. works fine. `AcrylicTaskViewTimeline: 12520383`

![Untitled1](https://github.com/valinet/ExplorerPatcher/assets/92948468/1dc05bbd-adcb-4d97-bc7e-1080051df6b5)

Here is the full list, just in case it helps with feature development or bug fixes:
[19045.4096.txt](https://github.com/valinet/ExplorerPatcher/files/14335378/19045.4096.txt)

**Comment by Amrsatrio** [comment]
Very interesting. This feature flag also appeared on latest Canary so might aswell make a new release to buy us some more time for this. Unfortunately I think this can only continue to exist on Windows 11 afterwards if MS decided to force that feature flag to be on :(

**Comment by Meshy24** [comment]
This worked for me! Thanks man!

**Comment by copytac** [comment]
I just found i need switch the taskbar style to win10 default, in win10 ep mod, the weather will missing.
